### PR TITLE
Add an index.html page that showcases all of the styling options

### PR DESCRIPTION
**Pull request by josephwegner**
This adds a styles page that showcases all of our various styles, how they work, and how they can be used. It should be used as a resource when diving into `keen-css` usage.

You can test this live at https://keen.github.io/keen-css/

TODO:
- [x] max-widths
- [x] overflows
- [x] opacity
- [x] rotation
- [x] hover effects
- [ ] ~tables~
- [x] text decoration
- [ ] ~typography~
- [ ] ~visibility~
- [x] white-space
- [x] vertical-align
- [x] pointers
- [x] z-index

screen shot 2017-04-19 at 4 58 01 pm
screen shot 2017-04-19 at 4 58 15 pm
screen shot 2017-04-19 at 4 58 25 pm
screen shot 2017-04-19 at 4 58 36 pm

### CODE REVIEW

**Comment by jacklittleton** [comment]
This is pretty much just a nit, but the ToC and content don't match for Opacity, Spacing, and Padding:

image 2017-04-26 at 1 32 24 pm public

**Comment by jacklittleton** [comment]
Potential copypasta issue in the Height section:

image 2017-04-26 at 1 40 29 pm public

**Comment by jacklittleton** [comment]
Typo in the Border section:

> And finally, border widths are set on a scal of 1-5

"scal" -> "scale"

**Comment by jacklittleton** [comment]
`.shadow-hover` doesn't seem to do anything for me on Chrome.

**Comment by sfalkoff** [comment]
Suggestion: align text in https://keen.github.io/keen-css/#spacing

screen shot 2017-04-26 at 3 00 41 pm

**Comment by sfalkoff** [comment]
Suggestion: reduce amount of Lorem ipsum copy and add whitespace between columns https://keen.github.io/keen-css/#floating

screen shot 2017-04-26 at 3 11 31 pm

File: index.html [hunk 1]
```diff
@@ -0,0 +1,1352 @@
...
+        <h3 class="mt5 f2" id="overflow">
+          Overflow
+          <span class="bg-light-gray code f7 pa1 b1">.overflow-scroll</span>
+        </h3>
+
+        <div class="flex flex-row flex-wrap overflow-hidden">
+          <div class="w-25-ns w-100 h4 pr2 pb2 mb4">
+            <span class="bg-light-gray code f7 pa1 b1">.overflow-visible</span>
+            <div class="pa2 mt2 h-100 ba bw2 b--blue overflow-visible">
+              Here is some text that is does not fit in the div containing it.
```
Comment (sfalkoff) [inline index.html]: Extra `is` in this sentence (used in each overflow example):

 "Here is some text that ~~is~~ does not fit in the div containing it."

**Comment by sfalkoff** [comment]
Nice sidebar nav! A non-essential but potentially useful addition may be to make all of the headings into links for easy copy/paste sharing. Something like https://keen.io/docs/streams/#data-enrichment.
 
screen shot 2017-04-26 at 3 41 24 pm

**Comment by josephwegner** [comment]
Thanks for the feedback everyone! I have fixed all of the things.

@sfalkoff I like the nav link idea, although I'm going to punt on it for now. I expect to improve the layout quite a bit when I add components, and will likely include that.

**Review by jacklittleton — APPROVED** [review]

**Pull request merged**